fetch_and_alert.py

```diff
@@ -1,158 +1,99 @@
-      
-      # Run Detection
-      results = DetectMD.load_and_run_detector_batch(image_paths, detector_model, log_dir,
-                                                     confidence_threshold, checkpoint_frequency, [])
-      # Parse results
-      df = FileManagement.parseMD(results)
-      # filter out all non animal detections
-      
-      if not df.empty:
-          animalDataframe, otherDataframe = FileManagement.filterImages(df)
-          # run classifier on animal detections if there are any
-          if not animalDataframe.empty:
-              # create generator for images
-              generator = ImageCropGenerator.GenerateCropsFromFile(animalDataframe)
-              # Run Classifier
-              predictions = model.predict_generator(generator, steps=len(generator), verbose=1)
-              # Parse results
-              maxDataframe = FileManagement.parseCM(animalDataframe, None, predictions, classes)
-              # Creates a data frame with all relevant data
-              cougars = maxDataframe[maxDataframe['class'].isin(targets)]
-              # drops all detections with confidence less than threshold
-              cougars = cougars[cougars['conf'] >= confidence_threshold]
-              # reset dataframe index
-              cougars = cougars.reset_index(drop=True)
-              # Sends alert for each cougar detection
-              for idx in range(len(cougars.index)):
-                  label = cougars.at[idx, 'class']
-                  prob = cougars.at[idx, 'conf']
-                  img = Image.open(cougars.at[idx, 'file'])
-                  draw_bounding_box_on_image(img,
-                                             cougars.at[idx, 'bbox2'], cougars.at[idx, 'bbox1'],
-                                             cougars.at[idx, 'bbox2'] + cougars.at[idx, 'bbox4'],
-                                             cougars.at[idx, 'bbox1'] + cougars.at[idx, 'bbox3'],
-                                             expansion=0,
-                                             use_normalized_coordinates=True,)
-                  imageBytes = BytesIO()
-                  img.save(imageBytes, format=img.format)
-                  smtp_server = smtp_setup(username, password, host)
-                  sendAlert(label, prob, imageBytes, smtp_server, username, to_emails)
-              # Write Dataframe to csv
-              cougars.to_csv(f'{log_dir}dataframe_{datetime.now().strftime("%m-%d-%Y_%H:%M:%S")}')
-
-
-def run_emails():
-    # Gets a list of attachments from unread emails from bigfoot camera
-    mail = imap_setup(host, username, password)
-    global timestamp
-    print('Starting Email Fetcher')
-    images = extractAttachments(fetch_emails(mail, from_emails, timestamp), mail, config_file)
-    print('Finished Email Fetcher')
-    print('Starting Detection')
-    detect(images)
-    print('Finished Detection')
+CHECKIN_INTERVAL = CONFIG['checkin_interval']
 
 
 def run_scraper():
```

Comment:
I think we ought to rename this function (and change the relevant print statements) to reflect that we are no longer web scraping to get the images but accessing them through the StrikeForce API.